src/Stratis.Bitcoin.Networks/StratisMain.cs

```diff
@@ -84,9 +84,17 @@ public StratisMain()
 
             var bip9Deployments = new StratisBIP9Deployments()
             {
-                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(2,
+                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(2, 
                     new DateTime(2018, 12, 1, 0, 0, 0, DateTimeKind.Utc),
-                    new DateTime(2019, 12, 1, 0, 0, 0, DateTimeKind.Utc))
+                    new DateTime(2019, 12, 1, 0, 0, 0, DateTimeKind.Utc)),
+
+                [StratisBIP9Deployments.CSV] = new BIP9DeploymentsParameters(3, 
+                    new DateTime(2019, 10, 1, 0, 0, 0, DateTimeKind.Utc),
+                    new DateTime(2020, 10, 1, 0, 0, 0, DateTimeKind.Utc)),
+
+                [StratisBIP9Deployments.Segwit] = new BIP9DeploymentsParameters(4, 
```

Comment:
Hmm, what about the bech32 encoders? When those are enabled I think we should choose a different prefix too.

Reply:
Changed to strat/tstrat